docs/source/source_code/source_code.rst

```diff
@@ -89,3 +96,10 @@ Source
    :glob:
 
    src/platform/windows/*
+
+.. toctree::
+   :caption: src/platform/windows/display_device
+   :maxdepth: 1
+   :glob:
+
+   src/platform/windows/display_device/*
```

Comment:
I've tried to do it, but it will then complain about files not included in the toc tree. I can modify the glob, but then the others are not included :/